styles/AdvancedEditor.css

```diff
@@ -639,10 +641,12 @@ span.comment-marker {
   margin-top: 0;
 }
 
-.author-notes > .block-section_title > .block,
-.keywords > .block-section_title > .block {
+.author-notes > .block-section_title > .block, .keywords > .block-section_title > .block {
   align-items: center;
 }
+.ProseMirror > .keywords > .block-section_title > h1, .ProseMirror > .selected-suggestion > .keywords > .block-section_title > h1{
```

Comment:
this update should be in https://github.com/Atypon-OpenSource/manuscripts-body-editor/blob/681263972bdf12e8e7fd840bc1ea50146042432c/styles/AdvancedEditor.css#L528

Reply:
actually, I can't update that selector with the mentioned one since this will affect the style of title when it's selected and many other things as well !

![image](https://github.com/user-attachments/assets/f1555754-1fdc-4732-8db6-78038739d5ec)

it's not a big deal anyway, I just wanted to add some space between the title and keywords since adding the requested border reduce that space .. I believe adding some margins to the keyword itself like before will be the best solution .. otherwise we can forget this :

.keyword-group .keyword {
  border: 1px solid #C9C9C9;
margin-top:10px
}

Reply:
this ticket should fix this issue: 
![Screenshot from 2024-07-16 01-26-04](https://github.com/user-attachments/assets/114f0a16-d2be-4272-b0b9-2889d4e0bffa)

when the keyword element get selected font size will change, it should not!

or is this ticket just about updating border for keyword?

Reply:
this is how all titles are being styled in the editor "blank editor" not only the keywords, you can check other sections like abstract ...etc 
and that was already discussed with vip and panos and there is a plan to improve this but not on this ticket. since there are alot of styling issues in the blank articles that need a design work .